Dockerfile.awscli

```diff
@@ -1,4 +1,4 @@
-FROM imione/awscli-local
+FROM mikesir87/aws-cli:latest
```

Comment:
unrelated, right?

Reply:
awscli-local is the localstack wrapper for aws cli. I'm now just using aws-cli.